### PR TITLE
Fix: Ensure background image covers full viewport

### PR DESCRIPTION
**Pull request by shadabkhan16**
## Description
Background image to fit the screen.

- What does this PR do?
- Why is it needed?

## Screenshots
ScreenRecording2024-12-06at3 03 30PM-ezgif com-video-to-gif-converter

### Desktop:
[Add screenshot here]

### Mobile:
[Add screenshot here]

## Approach
Briefly explain the approach taken to implement the changes. 

- How did you solve the problem?
- Any particular design patterns or optimizations?

## Issue Number (if any)
Link to the issue this PR addresses, if applicable:

Fixes #[issue_number]

## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/game_frontend/src/pages/Dashboard/layout.tsx [hunk 1]
```diff
@@ -11,6 +11,9 @@ const Layout = () => {
       className="exchange"
       style={{
         backgroundImage: 'url("/assets/images/space-exchange-bg.webp")',
+        width: '100vw',           // Full viewport width
+        height: '100vh',         // Full viewport height
```
Comment (ArjunQBTech) [inline src/game_frontend/src/pages/Dashboard/layout.tsx:15]: set `min-height : 100vh` and `height : auto`

File: src/game_frontend/src/pages/Dashboard/layout.tsx [hunk 2]
```diff
@@ -11,6 +11,9 @@ const Layout = () => {
...
+        overflow: 'hidden',     // Ensures no extra space
```
Comment (ArjunQBTech) [inline src/game_frontend/src/pages/Dashboard/layout.tsx:16]: Is there a reason for `overflow : hidden`

**Review by ArjunQBTech — CHANGES_REQUESTED** [review]

File: src/game_frontend/src/pages/Dashboard/layout.tsx (hunk 2 above)
Comment (shadabkhan16) [reply src/game_frontend/src/pages/Dashboard/layout.tsx:16]: Not  using  ```overflow : hidden```  scroll bar will visible

File: src/game_frontend/src/pages/Dashboard/layout.tsx (hunk 1 above)
Comment (shadabkhan16) [reply src/game_frontend/src/pages/Dashboard/layout.tsx:15]: Screen Recording 2024-12-06 at 4 54 02 PM
 If  follow your suggestion

**Comment by shadabkhan16** [comment]
This background image is same for all pages

File: src/game_frontend/src/pages/Dashboard/layout.tsx (hunk 2 above)
Comment (shadabkhan16) [reply src/game_frontend/src/pages/Dashboard/layout.tsx]: ScreenRecording2024-12-07at1 24 46PM-ezgif com-video-to-gif-converter

File: src/game_frontend/tailwind.config.js [hunk 3]
```diff
@@ -26,5 +26,34 @@ export default {
       ...defaultTheme.screens
     }
   },
-  plugins: [],
+  plugins: [
+    function ({ addUtilities }) {
```
Comment (ArjunQBTech) [inline src/game_frontend/tailwind.config.js:30]: Add this plugin by making a folder in utils directory : 
`utils/tailwind-plugins/index.tsx`

**Review by ArjunQBTech — CHANGES_REQUESTED** [review]

**Review by ArjunQBTech — APPROVED** [review]

**Review by ArjunQBTech — APPROVED** [review]

**Pull request merged**